setup.cfg

```diff
@@ -1,5 +1,6 @@
 [tool:pytest]
 addopts = -s
+timeout = 300
```

Comment:
are we sure this is working? one of the ubuntu tests is running for 1.5 hours now